src/palace/manager/api/sirsidynix_authentication_provider.py

```diff
@@ -153,6 +166,12 @@ def __init__(
         self.sirsi_disallowed_suffixes = library_settings.library_disallowed_suffixes
         self.sirsi_library_id = library_settings.library_id
 
+        # Check if patrons should be blocked based on SIP status
+        if settings.patron_status_block:
+            self.patron_status_should_block = True
+        else:
+            self.patron_status_should_block = False
```

Comment:
Minor: Since `patron_status_block` is already a `bool`, it probably makes sense to collapse this down to:

```suggestion
        # Check if patrons should be blocked based on SIP status
        self.patron_status_should_block = settings.patron_status_block:
```